### PR TITLE
Top nav

### PR DESCRIPTION
**Pull request by jonathandick**

### CODE REVIEW

File: src/banner/patient-banner.component.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default function PatientBanner(props: PatientBannerProps) {
   ] = useCurrentPatient();
 
   return (
-    <aside className={styles.patientBanner}>
+    <div className={styles.patientBanner}>
```
Comment (fatmali) [inline src/banner/patient-banner.component.tsx]: why did you remove the `aside` here? the `<aside>` basically identifies the content that is related to the primary content of the web page but does not constitute the main intent of the primary page.

File: src/chart-review/chart-review.component.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import LevelTwoRoutes from "./level-two-routes.component";
+import styles from "./chart-review.css";
+
+export default function ChartReview(props: any) {
+  const [selected, setSelected] = React.useState();
+  let history = useHistory();
+  const [lastRoute, setLastRoute] = React.useState(history.location.pathname);
+  const [paths, setPaths] = React.useState({
+    summaries:
+      history.location.pathname.indexOf("summaries/") > -1
```
Comment (fatmali) [inline src/chart-review/chart-review.component.tsx]:
```suggestion
      history.location.pathname.includes("summaries/")
```

File: src/chart-review/chart-review.component.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        ? history.location.pathname
+        : ""
+  });
+
+  let { patientUuid } = useParams();
+
+  React.useEffect(() => {
+    const p = paths;
```
Comment (fatmali) [inline src/chart-review/chart-review.component.tsx]: why not just use paths directly? why reassign it to `p`?

File: src/root.component.tsx [hunk 4]
```diff
@@ -5,53 +5,45 @@ import PatientChartOverview from "./summaries/overview/patient-chart-overview.co
 import PatientBanner from "./banner/patient-banner.component";
 import LevelTwoRoutes, {
   levelTwoRoutes
-} from "./summaries/level-two-routes.component";
-import { Breadcrumbs } from "./breadcrumbs/breadcrumbs.component";
+} from "./chart-review/level-two-routes.component";
 import Sidebar from "./sidebar/sidebar.component";
 import WorkspaceWrapper from "./workspace/workspace-wrapper.component";
+import ChartReview from "./chart-review/chart-review.component";
+import { blockStatement } from "@babel/types";
```
Comment (fatmali) [inline src/root.component.tsx]: is this used?

File: src/summaries/summaries-nav.component.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+  /*
+  React.useEffect(() => {
+    navItems.map((item, index) => {
+      if (item.path === history.location.pathname) {
+        setSelected(index);
+      }
+    });
+  }, []);
+  */
```
Comment (drizzentic) [inline src/summaries/summaries-nav.component.tsx]: We should consider removing any unused/commented code.

File: src/summaries/summaries-nav.component.tsx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  ];
+
+  let i = 0;
+  navItems.forEach((item, index) => {
+    if (item.path === history.location.pathname) {
+      i = index;
+    }
+  });
+
+  const hasPath = item => item.path.indexOf(history.location.pathname) > -1;
```
Comment (drizzentic) [inline src/summaries/summaries-nav.component.tsx]: For consistency, we might want to use `.includes()` in place of `.indexOf()`

File: src/summaries/summaries-nav.component.tsx [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+      path: `/patient/${patientUuid}/chart/summaries/hiv`
+    }
+  ];
+
+  let i = 0;
+  navItems.forEach((item, index) => {
+    if (item.path === history.location.pathname) {
+      i = index;
+    }
+  });
```
Comment (drizzentic) [inline src/summaries/summaries-nav.component.tsx]: Looks like this code is not being used.

File: src/banner/patient-banner.component.tsx (hunk 1 above)
Comment (jonathandick) [reply src/banner/patient-banner.component.tsx]: This was inadvertent. I think I removed it when trying to figure out layout issues and forgot to put it back.

File: src/chart-review/chart-review.component.tsx (hunk 2 above)
Comment (jonathandick) [reply src/chart-review/chart-review.component.tsx]: Done.

File: src/chart-review/chart-review.component.tsx (hunk 3 above)
Comment (jonathandick) [reply src/chart-review/chart-review.component.tsx]: No good reason. Changed.

File: src/root.component.tsx (hunk 4 above)
Comment (jonathandick) [reply src/root.component.tsx]: No, not sure how it got there. Removed.

File: src/summaries/summaries-nav.component.tsx (hunk 5 above)
Comment (jonathandick) [reply src/summaries/summaries-nav.component.tsx]: gone

File: src/summaries/summaries-nav.component.tsx (hunk 7 above)
Comment (jonathandick) [reply src/summaries/summaries-nav.component.tsx]: It gets used on the initial render of the component. 
There are three scenarios I've been trying to find ways (somewhat unsuccessfully) to account for:

1. The user navigates from outside patient chart to /chart. In this case, there should be no tabs selected. 
2. The user directly navigates to a url like : /chart/summaries/overview, in which case the summaries and overview tabs should be selected (this would be handled by the above code) 
3. The user is been navigating within a chart. For example, a user clicks on summaries --> overview, then clicks on the (main nav).orders and then clicks back to (main nav).summaries. In this case, I want the last summary tab that the user was on to be displayed. (this is the trickiest scenario).

**Comment by jonathandick** [comment]
I closed this PR in favor of https://github.com/openmrs/openmrs-esm-patient-chart/pull/107 (in which I probably did a sub-optimal merge with master)
